src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const x = Big(numberOne);
+  const y = Big(numberTwo);
+
+  switch (operation) {
+    case '-':
+      return x.minus(y).toNumber();
+    case '+':
+      return x.plus(y).toNumber();
+    case 'x':
+      return x.times(y).toNumber();
+    case '/':
+      return x.div(y).toNumber();
+    case '%':
+      return x.div(100).toNumber();
```

Comment:
According to the project requirement, I think you need to use `modifier` here.

![image](https://user-images.githubusercontent.com/31897434/104402184-071fb900-5556-11eb-933f-57d1965c8e5d.png)
https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376018/

Reply:
i did not understand what it meant

Reply:
@misterpaul4,

It is one of the `Big.js` features.
kindly read more 👉 https://mikemcl.github.io/big.js/

Example: `x.mod(y).toString()`